index.html

```diff
@@ -2225,6 +2225,28 @@ <h2>The <dfn>record type</dfn> string</h2>
   </div>
   </section>
 
+  <section><h3>Aborting pending scan operation</h3>
+  <div>
+    To attempt to <dfn>abort a pending scan operation</dfn>,
```

Comment:
Shall we run "abort a pending scan operation" as well when NFC is suspended?
Same way we do for Write?

Reply:
Good point. We did, but now we can replace 2 steps with one. Coming.

Reply:
Also found another error in aborting scan, fixed.